src/NodaTime.Demo/LocalDateDemo.cs

```diff
@@ -51,5 +51,23 @@ public void AtMidnight()
             LocalDateTime dateTime = Snippet.For(date.AtMidnight());
             Assert.AreEqual(new LocalDateTime(2010, 6, 16, 0, 0, 0), dateTime);
         }
+
+        [Test]
+        public void Max()
+        {
+            LocalDate date = new LocalDate(2010, 6, 16);
```

Comment:
I'd suggest renaming these to something like `earlyJune` and `lateJune` (possibly making them the 5th and 25th of the month), and save the result into `max` or `result`.

Reply:
Any point to be in June?

Reply:
Like this?
```csharp
[Test]
public void Max()
{
    LocalDate earlyJune = new LocalDate(2010, 6, 5);
    LocalDate lateJune = new LocalDate(2010, 6, 10);
    var max = LocalDate.Max(earlyJune, lateJune);
    LocalDate dateTime = Snippet.For(max);
    Assert.AreEqual(new LocalDate(2010, 6, 16), dateTime);
}
```

Reply:
Not quite - the idea is to use `max` instead of `dateTime`. Currently that's not going to create a snippet for anything - the expression in `Snippet.For` has to be the method that we're trying to document. I meant:

```csharp
[Test]
public void Max()
{
    LocalDate earlyJune = new LocalDate(2010, 6, 5);
    LocalDate lateJune = new LocalDate(2010, 6, 25);
    var max = Snippet.For(LocalDate.Max(earlyJune, lateJune));
    Assert.AreEqual(new LocalDate(2010, 6, 16), max);
}
```

Note that I've updated `lateJune` to be June 25th, as June 10th isn't very late in June :)
(There's nothing specific about June here - I'm just making the name reflect the actual value in a simple way.)

Reply:
about june 10th i forgot to update, thanks for help!